src/SearchAnime.js

```diff
@@ -0,0 +1,47 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
+
+
+const SearchedAnime = () => {
+
+    const { animeSearched } = useParams();
+    const [anime, setAnime] = useState([]);
+
+    useEffect( () => {
+        axios({
+            url: 'https://api.jikan.moe/v4/anime',
+            method: 'GET',
+            dataResponse: 'json',
+            params: {
+                q: animeSearched,
+            }
+        }).then( (res) => {
+            setAnime(res.data.data);
+        })
+    }, [animeSearched])
+
+    return(
+        <section>
+            <Link to={'/'}>
+                <h3>Home</h3>
+            </Link>
+            <section className="anime">
+            {
+                anime.map( (anime) => {
```

Comment:
Good job on getting the search functionality to work with a text input. Def tricky cause there are case sensitivity issues and spacing but you took care of it!  I like how it returns multiple titles when I search up One Piece (the amount of movies and OVAs is insanity lol)